### DISCUSSION TITLE
OSCAL 1.0.6 Release

### DISCUSSION THREAD

**Discussion by aj-stein-nist** [usnistgov/OSCAL#1832] [Releases]
## What's Changed
* Complete pre-release in 1.0.5 and correct datatype regex for https://github.com/usnistgov/OSCAL/issues/1703 by @aj-stein-nist in https://github.com/usnistgov/OSCAL/pull/1736
* Add updated Metaschema 0.9.0 release in #1820 to fix bugs introduced in the 1.0.5 pre-release, reported, and fixed by community contributors @kylelaker and @Telos-sa in #1773 and #1774.
* Adjustments to align with Metaschema schema bug fixes by @david-waltermire-nist in https://github.com/usnistgov/OSCAL/pull/1805

**Full Changelog**: https://github.com/usnistgov/OSCAL/compare/v1.0.5...v1.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/OSCAL/releases/tag/v1.0.6'>OSCAL 1.0.6 Release</a>.</em>

**Comment by FunkyARUI** [comment]
Hi there, thanks for updating! I have a question about schema update, seems this 1.0.5 -> 1.0.6 doesn't make any schema change for POAM, which is to say, today's POAM schema(https://pages.nist.gov/OSCAL/reference/latest/plan-of-action-and-milestones/json-outline/) would stay the same, is it right?

**Reply by aj-stein-nist** [reply]
There were bugs in the schema in 1.0.5 that incorrectly changed which fields were required and the strictness of data types (like timestamps) from 1.0.4, but the structure of the POAM did not change separate of those issues, you are correct.

**Reply by FunkyARUI** [reply]
Thanks for clarifying!

**Reply by aj-stein-nist** [reply]
You're welcome, @FunkyARUI. If you want to know how to evaluate the differences and [test a new release, more info is here](https://pages.nist.gov/OSCAL/learn/tutorials/general/releases/).
